CriticalMass/MapInfoView.swift

```diff
@@ -0,0 +1,81 @@
+//
+//  MapInfoView.swift
+//  CriticalMaps
+//
+//  Created by Leonard Thomas on 13.12.19.
+//  Copyright © 2019 Pokus Labs. All rights reserved.
+//
+
+import UIKit
+
+class MapInfoView: UIView, IBConstructable {
+    struct Configuration {
+        enum Style: String {
+            case alert
+            case info
+        }
+
+        var title: String
+        var style: Style
+    }
+
+    @IBOutlet private var imageView: UIImageView!
+    @IBOutlet private var label: UILabel!
+
+    @objc
+    dynamic var mapInfoForegroundColor: UIColor = .black {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    @objc
+    dynamic var mapInfoBackgroundColor: UIColor = .white {
+        didSet {
+            updateStyle()
+        }
+    }
+
+    private var configuration: Configuration?
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        layer.cornerRadius = 18
+        layer.shadowColor = UIColor.black.cgColor
+        layer.shadowRadius = 4
+        layer.shadowOpacity = 0.2
+        layer.shadowOffset = CGSize(width: 0, height: 2)
```

Comment:
maybe this can be a setupLayer func like
```swift
private func setup(_ layer: CALayer) {
    layer.cornerRadius = 18
    layer.shadowColor = UIColor.black.cgColor
    layer.shadowRadius = 4
    layer.shadowOpacity = 0.2
    layer.shadowOffset = CGSize(width: 0, height: 2)
}
```

and then 

```swift
setup(layer)
```

Reply:
Apparently,  the navigation overlay uses the same shadow and corner radius parameters. I moved the code to an extension